admin/option_values.php

```diff
@@ -86,112 +108,87 @@
 
     <!-- body //-->
     <div class="container-fluid">
-      <div class="table-responsive">
-        <h1><?php echo HEADING_TITLE; ?></h1>
+
+        <h1><?= HEADING_TITLE; ?></h1>
```

Comment:
I'm liking the short-echo tags more and more, but when they're used, the final `;` in the associated PHP statement should not be included.  The same is true for the additional usages in this PR, please either revert the short-echo or remove the `;' in each place.